folly/portability/PThread.h

```diff
@@ -22,9 +22,18 @@
 
 #include <pthread.h>
 
+#if __has_include(<pthread_np.h>)
+#include <pthread_np.h>
```

Comment:
Which platform is this for?

Reply:
for all intents and purposes, FreeBSD: https://github.com/freebsd/freebsd/blob/master/include/pthread_np.h